src/views/includes/macros.njk

```diff
@@ -73,10 +73,27 @@
   {%- endif -%}
 {%- endmacro -%}
 
-{%- macro formatNationality(str) -%}
+{%- macro formatNationality(str, i18n) -%}
+  {%- set localeStr = i18n.nationalities[makeCamelCase(str)] -%}
   {%- set exceptions = ['of', 'the', 'and', 'citizen'] -%}
-  {%- set words = str.split(' ') -%}
+  {%- set words = localeStr.split(' ') -%}
   {%- set result = [] -%}
+
+  {# Temporary while waiting for welsh translations - need to handle 'WELSH -' keyword #}
```

Comment:
i wouldn't worry about preserving the WELSH in capitals, it is only temporary so no harm if it appears as "Welsh - " on the page, and cleans the code up a bit if it is removed

Reply:
I did this to get the tests to pass, this is why I included the note here so we can remove it once the real translations are added.

Reply:
ok I forgot about the tests.

I was also wondering about the line with concat(words[1]). I'm assuming that words[1] will be "-". Can it just be handled with the rest of the words in the for loop so you don't need to set result = result.concat(words[1])?